client/src/components/HomePage/HomePage.js

```diff
@@ -81,15 +74,16 @@ class HomePage extends React.Component {
                 className="card-img-top"
                 alt="buzz"
```

Comment:
This looks like where the random 'buzz' is coming from, since the image isn't loading; The alt should probably be a variable, or the image-class here from the card should probably be removed. 

If the className is coming from a package (card-img-top), it may be a matter of making sure it's imported into this component.